overlord/snapstate/snapstate_update_test.go

```diff
@@ -1408,6 +1408,8 @@ func (s *snapmgrTestSuite) TestParallelInstanceUpdateRunThrough(c *C) {
 		Version:     "services-snapVer",
 		PlugsOnly:   true,
 		InstanceKey: "instance",
+		// parallel installs should implicitly pass --unaliased
+		Flags: snapstate.Flags{Unaliased: true},
```

Comment:
this shouldn't be required as refreshes should preserve the current aliases setup?

Reply:
Yes, thank you for pointing this out, updated!